cdap-api-common/src/main/java/co/cask/cdap/api/data/format/StructuredRecord.java

```diff
@@ -58,6 +58,15 @@ public Schema getSchema() {
   }
 
   /**
+   * Get the fields in the record.
+   *
+   * @return the fields in the record.
+   */
+  public Map<String, Object> getFields() {
```

Comment:
This is an internal detail. should probably not expose it outside, especially just for tests.